### DISCUSSION TITLE
Faker import issue with Ember

### DISCUSSION THREAD

**Discussion by mtrynoha** [faker-js/faker#2931] [Q&A]
Hi!
I'm trying to use @faker-js/faker in my Ember app but encountering an issue.
Node version: 14
Ember CLI: 3.14
Faker: 8.4

In tests it runs perfectly and generates data as expected, however when I start the server locally (emebr serve), I'm getting this error in the web console:
`loader.js:247 Uncaught Error: Could not find module `@faker-js/faker` imported from `ember-frontend/tests/factories/user``

Here is the code of `ember-frontend/tests/factories/user`:
```
import FactoryGuy from 'ember-data-factory-guy';
import { faker } from '@faker-js/faker';

FactoryGuy.define('user', {
  default: {
    email: faker.internet.email(),
    firstName: faker.person.firstName(),
    lastName: faker.person.lastName(),
  }
});

```
I've ensured @faker-js/faker is installed as a dev dependency and tried various configuration adjustments without success. Any piece of advice would be greatly appreciated.

Thank you!

**Comment by ST-DDT** [comment]
Have you tried using faker as normal dependency temporarily?

If you use it at runtime you have to use it as dependency vs if you only need it during development/tests then you need it as a dev dependency.

Is our documentation confusing in that regard?

**Reply by mtrynoha** [reply]
Thanks for response.
Yes, I did try using it as normal dependency, but the result remains the same.
I don't use it at runtime, all the files that import faker are under tests directory.

**Reply by ST-DDT** [reply]
Sorry, I'm not familiar with ember so I'm unsure on how to help.

The closest I could guess is this article:
https://guides.emberjs.com/release/addons-and-dependencies/
I'm pretty sure you have seen it already, but I post it just in case.
